src/utilities/util_verify.c

```diff
@@ -43,12 +48,20 @@ verify_one(WT_SESSION *session, char *config, char *uri)
 {
     WT_DECL_RET;
 
-    if ((ret = session->verify(session, uri, config)) != 0)
+    ret = session->verify(session, uri, config);
+
+    /*
+     * The ENOENT error means that a specific checkpoint verification was requested and this
+     * checkpoint could not be found for the provided URI.
+     */
+    if (ret == ENOENT)
+        return (ret);
```

Comment:
Why do we not fallback on what's below?
```
    if (ret != 0)
        ret = util_err(session, ret, "session.verify: %s", uri);
```

Reply:
That's because if we provide some specific checkpoint, it is OK not to find this checkpoint in some tables. The main rule here is that this checkpoint should exist for at least 1 table.

But here it can be unclear, and I found couple of bugs in this place, so I hope the new logic brought more clarity here.